lib/NodeHelpers.js

```diff
@@ -131,17 +160,24 @@ function getLocalMatrix4(node, result) {
     rotation.w = rotationArray[3];
 
     return Matrix4.fromTranslationQuaternionRotationScale(translation, rotation, scale, result);
-}
+};
 
 var packedParametersScratch = {
     meshes : undefined,
     parameters : undefined,
     primitiveFunction : undefined
 };
 
-// Perform an operation on each primitive in a scene.
-// PrimitiveFunction should expect primitive, a meshPrimitiveID, some parameters, and the node itself
-function forEachPrimitiveInScene(gltf, scene, primitiveFunction, parameters) {
+/**
+ * Perform an operation on each primitive in a scene.
+ * primitiveFunction has the signature: function(primitive, meshPrimitiveId, parameters, node).
+ *
+ * @param {Object} gltf A javascript object holding a glTF hierarchy.
+ * @param {Object} scene The glTF scene to traverse.
+ * @param {Function} primitiveFunction The function to apply to each primitive.
+ * @param {Object} parameters Parameters to pass to primitiveFunction.
+ */
+NodeHelpers.forEachPrimitiveInScene = function(gltf, scene, primitiveFunction, parameters) {
```

Comment:
This is unrelated to this class and could be its own file.

Also, I am not confident that this class in general is the right design compared to separate functions.  Combing loosely related functions into a "helper" class is a typical code "smell."  We can leave it for now (would love to see a name without "helper" in it though), but I expect we will change this as we introduce whatever the jQuery for glTF ends up being.